### ISSUE TITLE
[Suggestion] Apply colorscheme to Sauron's windows/panels

### ISSUE THREAD

**Issue by tinkerng** [racket-tw/sauron#192]
**Is your feature request related to a problem? Please describe.**
Sauron's windows/panels have a white background even with dark themes/colorschemes active.

It's a small thing, users of light themes/schemes won't be affected and the others won't care much, but the contrast causes some strain after a while.

**Describe the solution you'd like**
Having Sauron inherit the background from globally define themes or DrRacket's colorscheme.

**Describe alternatives you've considered**
Keeping the project viewer hidden and using the terminal or the REPL for version control.

**Additional context**
Project viewer:

![sc1](https://user-images.githubusercontent.com/90969776/155592661-b86ed89c-def9-4142-9e2c-27fff3b2b707.jpg)

Version control:

![sc2](https://user-images.githubusercontent.com/90969776/155592744-9b68c5b0-2fb2-488d-963b-f876db75fee1.jpg)

**Comment by dannypsnl** [comment]
This definitely would be good, I think it's related to [`color-prefs:register-info-based-color-schemes`](https://docs.racket-lang.org/framework/Color_Prefs.html#%28def._%28%28lib._framework%2Fmain..rkt%29._color-prefs~3aregister-info-based-color-schemes%29%29), the idea is if
1. framework:basic-canvas-background
2. framework:default-text-color

are provided, sauron will try to use it?

**Comment by dannypsnl** [comment]
or #194 will solve this problem(by throwing it to others, Jee!)

**Comment by tinkerng** [comment]
Honestly, I don't know enough to make any good suggestions about the implementation, but seeing #194, files-viewer is a good example of what I meant, it inherits both the GTK theme (the path in the top) and DrRacket's colorscheme (the main area).

**Comment by dannypsnl** [comment]
Yes, and those files-viewer didn't give in few years ago are implemented now, so I think bundle it is good now

**Comment by dannypsnl** [comment]
short note: extracts `color%` objects from the configuration
```racket
#lang racket
(require framework)

(define-values (a-set b-set)
  (color-prefs:get-color-scheme-names))

(for/list ([color-name (in-set a-set)])
  (cons color-name
        (color-prefs:lookup-in-color-scheme
         color-name)))
```
